<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BmE Sonar</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../.designer/backup/bat.png</normaloff>../../../.designer/backup/bat.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>23</number>
        </property>
        <item>
         <widget class="PlotWidget" name="graphWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="holdMeasurementBox">
          <property name="text">
           <string>Hold measurements</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="measureButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Measure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MeasurementName"/>
        </item>
        <item>
         <widget class="QPushButton" name="SaveMeasurementButton">
          <property name="text">
           <string>Save Measurement</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QListWidget" name="measurementList"/>
        </item>
        <item>
         <widget class="QPushButton" name="plotDataButton">
          <property name="text">
           <string>Plot data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteDataButton">
          <property name="text">
           <string>Delete Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuData"/>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>Export Measurement</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Empty data buffer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
